material-ui-superselectfield/material-ui-superselectfield.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module "material-ui-superselectfield" {
+    import * as React from "react"
+
+    interface IMaterialUISuperSelectFieldProps {
```

Comment:
This is a great start but it's technically missing out on some of the different options that exist for `superselectfield` like `anchorOrigin`

I'd recommend looking at the property docs [here](https://github.com/Sharlaan/material-ui-superselectfield) to get the full list 🦅

Reply:
Will look into this today!